### PR TITLE
Sumlist transformer

### PR DESCRIPTION
**Pull request by bakatzir**
## Status
Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/18511

## Description
added a new sum list of numbers transformer

## Does it break backward compatibility?
   - No

## Must have
- [x] Tests
- [ ] Documentation - after cr
- [x] Code Review

### CODE REVIEW

File: Scripts/SumList/SumList.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+name: SumList
+script: '-'
+system: false
+tags:
+- transformer
+- number
+timeout: '0'
+type: python
+runas: DBotWeakRole
+runonce: false
```
Comment (yuvalbenshalom) [inline Scripts/SumList/SumList.yml]: please add demisto default docker image for python3

**Comment by idovandijk** [comment]
Bar please tell me once you make Yuval's changes so I can test it

**Comment by bakatzir** [comment]
@idovandijk you can test

File: Scripts/SumList/SumList.yml (hunk 1 above)
Comment (bakatzir) [reply Scripts/SumList/SumList.yml]: @yuvalbenshalom  done

**Comment by idovandijk** [comment]
Works great!
image

Just one question - why is the description "list to reverse"?
image

**Comment by bakatzir** [comment]
@kirbles19 you can review the files:
https://github.com/demisto/content/blob/ac5b8316021d46478bc192481d8a3711ca31c5d7/Scripts/SumList/CHANGELOG.md
https://github.com/demisto/content/blob/ac5b8316021d46478bc192481d8a3711ca31c5d7/Scripts/SumList/SumList.yml

**Comment by kirbles19** [comment]
@bakatzir Done.

**Pull request merged**